src/clarity-icons/clarity-icons.scss

```diff
@@ -71,6 +83,14 @@ clr-icon {
     &[dir="left"] {
         @include rotateSVGIcon(270deg);
     }
+    &[shape$=" flip-horizontal"],
```

Comment:
I like what @victormejia has done here. `[dir="horizontal"]` is confusing to me. But `[flip="horizontal"]` is very clear. So I think what Victor has is great as is...